go.mod

```diff
@@ -3,14 +3,17 @@ module github.com/crossplane/crossplane
 go 1.13
 
 require (
+	github.com/Azure/go-autorest/autorest/validation v0.2.0 // indirect
```

Comment:
Are these Azure and AWS changes expected? I'm guessing they relate to k8schain?

Reply:
@negz yep! We actually don't use them at build time (we exclude the functionality with build tags), so it is kind of unfortunate that they show up here.